src/Assets/stepDirectory.ts

```diff
@@ -0,0 +1,51 @@
+import questions, { type QuestionNames } from './questions';
+
+export const startingQuestionNumer = 2;
+const defaultStepDirectory: QuestionNames[] = [
+  'zipcode',
+  'healthInsurance',
+  //the hhSize and hhData have to be consecutive
+  'householdSize',
+  'householdData',
+  'hasExpenses',
+  'householdAssets',
+  'hasBenefits',
+  'acuteHHConditions',
+  'referralSource',
+  'signUpInfo',
+];
+const referrerStepDirectory: QuestionNames[] = [
+  'zipcode',
+  'healthInsurance',
+  //the hhSize and hhData have to be consecutive
+  'householdSize',
+  'householdData',
+  'hasExpenses',
+  'householdAssets',
+  'hasBenefits',
+  'acuteHHConditions',
+  'signUpInfo',
+];
+
+export function getStepDirectory(referrer: string | undefined) {
+  if (referrer) {
+    // if referrer is not undefinded or '' skip the referrer question
+    return referrerStepDirectory;
+  }
+  return defaultStepDirectory;
```

Comment:
What do you think about wrapping this other return statement in an `else` block?